src/Youtube.js

```diff
@@ -190,6 +190,14 @@ THE SOFTWARE. */
         playerVars.theme = this.options_.theme;
       }
 
+      // Allow undocumented options to be passed along via customVars
+      if (typeof this.options_.customVars !== 'undefined') {
+        var self = this;
```

Comment:
Oh yeah, that's much better; I always feel awful writing `var self = this;`. I'll push an update to this PR shortly. Thanks for the suggestion!